docs/sections/user_guide/yaml/components/ww3.rst

```diff
@@ -0,0 +1,32 @@
+.. _ww3_yaml:
+
+ww3
+===
+
+Structured YAML to configure WaveWatchIII as part of a compiled coupled executable is validated by JSON Schema and requires the ``ww3:`` block, described below.
+
+Here is a prototype UW YAML ``ww3:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/ww3.yaml
+
+UW YAML for the ``ww3:`` Block
+------------------------------
+
+namelist:
+^^^^^^^^^
+
+.. important:: The WaveWatchIII namelist file is provisioned by rendering an input template file containing Jinja2 expressions. Unlike namelist files provisioned by ``uwtools`` for other components, the WaveWatchIII namelist file will not be validated.Note that the non-namelist ``ww3_shel.inp`` file may not be used as the basis for the input template file.:
```

Comment:
```suggestion
.. important:: The WaveWatchIII namelist file is provisioned by rendering an input template file containing Jinja2 expressions. Unlike namelist files provisioned by ``uwtools`` for other components, the WaveWatchIII namelist file will not be validated.
```
I think the note about the bad `ww3_shel.inp` file makes better sense adjacent to the info about the good `ww3_shel.nml`, below.